### PR TITLE
Switch some tests to do sample override and drop decision maker

### PR DESCRIPTION
**Pull request by bantonsson**
## Description

Switch some header tests to check sample override and decision maker drop as well.

## Workflow

1. ⚠️⚠️ Create your PR as draft (we're receiving lot of PR, it saves us lot of time) ⚠️⚠️
2. Follows the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

> **_NOTE:_**  By default in PR only default scenario tests will be launched. Please refer to the [documentation](https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2866381467/CI+Workflow+Github+Actions) to run all scenarios in your PR if needed.

Once your PR is reviewed, you can merge it ! :heart:

### CODE REVIEW

**Comment by bantonsson** [comment]
Hey @cbeauchesne, do you know who should approve this when @Kyle-Verhoog is not around? I think we need more than one code owner for this.

**Comment by Kyle-Verhoog** [comment]
>  Hey @cbeauchesne, do you know who should approve this when @Kyle-Verhoog is not around? I think we need more than one code owner for this.

+1 to this. Can we maybe have apm-core be code owners? (Although I don't see a github team for all of them combined)

**Review by Kyle-Verhoog — APPROVED** [review]

**Pull request merged**
